src/test/jtx/impl/envconfig.cpp

```diff
@@ -38,9 +38,9 @@ setupConfigForUnitTests(Config& cfg)
     cfg.FEES.reference_fee = 10;
     cfg.FEES.account_reserve = XRP(200).value().xrp().drops();
     cfg.FEES.owner_reserve = XRP(50).value().xrp().drops();
-    cfg.FEES.extension_compute_limit = 4294967295;
-    cfg.FEES.extension_size_limit = 4294967295;
-    cfg.FEES.gas_price = 1000000;
+    cfg.FEES.extension_compute_limit = 1'000'000;
+    cfg.FEES.extension_size_limit = 100'000;
+    cfg.FEES.gas_price = 1'000'000;  // 1 XRP = 1,000,000 micro-drops
```

Comment:
1 XRP = 1,000,000 micro-drops? what is the difference between drop and micro-drops?

Reply:
Oops, typo - should be "1 drop = 1 million micro-drops", fixed